orttraining/orttraining/core/optimizer/graph_transformer_utils.cc

```diff
@@ -120,9 +124,10 @@ std::vector<std::unique_ptr<GraphTransformer>> GeneratePreTrainingTransformers(
           execution_provider, false /*skip_dequantize_linear*/, compatible_eps, excluded_initializers));
       transformers.emplace_back(std::make_unique<ReshapeFusion>(compatible_eps));
       transformers.emplace_back(std::make_unique<ConcatSliceElimination>(compatible_eps));
-#if defined(USE_CUDA) || defined(USE_ROCM)
-      transformers.emplace_back(std::make_unique<ComputationReductionTransformer>(compatible_eps));
```

Comment:
Why are we deleting this transformer?

Reply:
Oh This is a renaming., now it is called ComputeOptimizer